app/updaters/shopify/variant_updater.rb

```diff
@@ -0,0 +1,29 @@
+module Shopify
+  class VariantUpdater
+    def initialize(spree_variant_id:, variant_klass: Spree::Variant, variant_api: ShopifyAPI::Variant, attributor: Shopify::VariantAttributes)
+      @spree_variant = variant_klass.find(spree_variant_id)
+      @variant_api = variant_api
+      @attributor = attributor
+    end
+
+    def perform
```

Comment:
💥 might have comment on this change for the same file in another branch and this may change it. Nice.